src/components/SecondaryNav/SecondaryNav.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+/**
+ * Secondary navigation typically used under the header
+ *
+ * Must pass an array of NavLinks for the navigation bar.
+ * Each NavLink should contain an anchor with appropriate href, or a react
+ * router Link or equivelant.
+ */
+const SecondaryNav = ({ buttons, className }) => {
```

Comment:
SecondaryNav should have an `onNavClick` prop that fires whenever any SecondaryNavLink is clicked

Reply:
@dankolbman what do you think about having `buttons` prop value just be an array of objects 
```js
[{
title: String, 
onClick:Function, 
active: Boolean, 
class: String (optional)
}]
```
that just gets mapped internally to SecondaryNavLinks?

This will provide better encapsulation and constraints

Reply:
If we do that, we should revise the Header nav too.
I'm more a fan of having the Navs take children rather than props for their buttons and components. I guess I'm just more partial to the DOM approach than the js one.

Reply:
what is the DOM approach? if we allow anything to be passed to buttons it could lead to inconsistency

Reply:
The DOM approach as in using children instead of objects:
```
<NavBar>
  <NavLink />
  <NavLink />
</NavBar>
```

Reply:
@dankolbman again though, I think that's too much freedom. I can't see a use case where we need would want to add anything other than `NavLink` as children so why open up the possibility for inconsistency?

Reply:
You also have freedom to provide any shape of objects, too, unless you enforce constraints.
You can also put constraints on the types of children being passed in, so functionally it's not really any different.

Reply:
valid point